Create modding-angular-apps.md

davidroberts63 opened the pull request:
A talk about modding Angular apps via Chrome Extensions or even server side.

View the abstract here: https://github.com/davidroberts63/talks/pull/6/files

davidroberts63 committed:
Create modding-angular-apps.md

davidroberts63 committed:
Adding notes and points to cover

davidroberts63 committed:
Update modding-angular-apps.md

davidroberts63 committed:
Update modding-angular-apps.md

davidroberts63 committed:
Update modding-angular-apps.md

jagthedrummer commented:
Hey David, sorry for the delay getting back to you on this. Below are some of my thoughts. Feel free to incorporate or disregard as you see fit. I'm not familiar enough with the topic to give you very specific feedback on that, but I can point out places that seem rough to me, or that leave me feeling like there are unanswered questions about the treatment of the topic.

1) In the first paragraph (the abstract) I would try to turn the 3rd sentence into a "you centered" sentence instead of a "me centered" sentence. Maybe something like "By digging into the lesser known areas of Angular you can add features seamlessly." This isn't a huge thing, but in general I try to keep the abstract focused on the viewer (and benefits to them), and not on myself.

2) It seems like there's either too much or not enough discussion of Octopus deploy, depending on what you're actually planning to do with it. Judging from the rest of the text here it seems like Octopus is not a main focus of the talk and it could probably be eliminated from the pitch (even if you still mention it in the talk itself). If I'm reading this with my "conf organizer hat" on I'm thinking "Woah! Where does Octopus deploy come into play? Aren't we talking about modding an app that I don't own and therefore won't be deploying with Octopus?" On the other hand, if Octopus is central to the ideas of the talk then maybe there should be a little more info about it. And maybe the title should be something like "Modding Angular apps with Octopus deploy". I don't know enough about what you're doing to make that call one way or the other, but it just seems like a weird thing to throw in and, for me, it raises more questions than it answers.

3) At NerdBeers the other night I thought you said something about this being useful in the context of writing add-ons/plugins/whatever-they're-called-in-Angular-land, where you are providing some code that is intended to be used inside other Angular apps to modify their behavior. Is that right? If so it would be good to mention that, or to even make it the focus of the talk. If that's not the case you might want to describe a little bit more about the Chrome plugin side of things, or whatever other mechanism you have for modifying apps in the wild.

davidroberts63 commented:
This is wonderful feedback. I feel that you not being familiar with the topic at hand gives you a better general view of the abstract. Thank you. And yes Octopus is not the main point of the talk. It may be confusing with me having additional notes below the abstract. I'm putting a lot of extra text in that doc that doesn't apply to what would be submitted. However, you mentioning that is a good point I will keep in mind in the future.

jagthedrummer commented:
No problem, happy to help! For many CFPs you'll find that they want you to submit an abstract, which will be used in the program if your talk is selected, but they ALSO want you to include notes for the review committee and/or a pitch on why your talk should be selected. The abstract is kind of the "tease" that's meant to bring people in the door without giving away all the secrets, but in the notes/pitch area I like to include a rough outline of the entirety of the talk so that the committee can see that I know what I'm talking about and that I have a reasonable outline of what to cover. So, I was looking at all of this as the first paragraph being the abstract, and the rest of it being the notes/pitch.

davidroberts63 committed:
More 'you'/viewer based

Also pointing out Angular 1.5 and specifically pointing out what things will be seen.

Though I'm starting to think I may want to break out a completely separate talk focused just on Chrome Extension 101

davidroberts63 commented:
Ahh, that makes sense. Thank you again.
